File: playground_series/s3e6/jl/baseline.jl
using CSV
using DataFrames
using Statistics

trn = CSV.read("./playground_series/s3e6/data/train.csv", DataFrame)
tst = CSV.read("./playground_series/s3e6/data/test.csv", DataFrame)

pred = mean(trn.:price)

sub = DataFrame(
    id = tst.:id,
    price = pred
)

CSV.write("./playground_series/s3e6/submissions/baseline.csv", sub)

File: playground_series/s4e1/py/baseline.py
import polars as pl
import statistics
import numpy as np

trn = pl.read_csv("./playground_series/s4e1/data/train.csv")
tst = pl.read_csv("./playground_series/s4e1/data/test.csv")

trn.head()

trn.shape

# get target
y = trn["Exited"]

# write code to take the mean of y
p = statistics.mean(y)

# encode predictions in a new dataframe
sub = pl.DataFrame({"id": tst["id"], "Exited": p})

# write to csv
sub.write_csv("./playground_series/s4e1/submissions/baseline.csv")
